### DISCUSSION TITLE
Android, installed plugin fails to launch/ show GUI

### DISCUSSION THREAD

**Discussion by cmgriffith** [zsviczian/obsidian-excalidraw-plugin#413] [General]
Could be an android version issue? Worked perfect for me on Android 12 mobile, whereas on Android 10 tablet (samsung tab s4) plugin does nothing. Installs and enables OK, but creating excalidraw page yields a blank screen, with Obsidian 1.0.5 and plugin 1.5.30

What extra information can I provide eg. android logs/ software information? Attatched is screenshot after creating excalidraw page
![SmartSelect_20220204-164205_Obsidian](https://user-images.githubusercontent.com/86885235/152568506-61858003-8641-4b20-8d76-b367ab0c642e.jpg)
.

**Comment by MystreAqui** [comment]
This happens to me too

**Reply by sunsoy** [reply]
https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/181

for me it was a webview issue. fixed after updating webview, give that a try, see linked thread

**Reply by MystreAqui** [reply]
Thanks bro , i updated webview for Android and its working now.  Bless Up😎

**Reply by sunsoy** [reply]
thank @zsviczian , I merely repeat his guidance

**Reply by MystreAqui** [reply]
Will do
